src/components/atoms/CheckBox.tsx

```diff
@@ -0,0 +1,14 @@
+import { Checkbox } from "@chakra-ui/react";
+import { memo, VFC } from "react";
+import { CheckBoxProps } from "../../types/PropsTypes";
+
+type Props = CheckBoxProps;
+
+export const CheckBox: VFC<Props> = memo((props: Props) => {
+	const { children, onChange, isChecked, colorScheme, size } = props;
+	return (
+		<Checkbox colorScheme={colorScheme} onChange={onChange} isChecked={isChecked} size={size}>
+			{children}
+		</Checkbox>
+	);
+});
```

Comment:
propsの型定義ですが、以下のようにすると拡張性が高くなり、記述もだいぶ楽になりますのでオススメです〜

`PropsTypes.ts`でpropsの型定義をまとめているようですが、コンポーネントの数に応じてこのファイルが肥大化していき、管理が大変になりそうです👀

propsの型定義はコンポーネントファイル内に記述し、必要に応じて`export type Props`したりすると良いかもです

```suggestion
import { Checkbox, CheckboxProps } from "@chakra-ui/react";
import { memo, ReactNode, VFC } from "react";

type Props = CheckboxProps & {
	children: ReactNode;
};

export const CheckBox: VFC<Props> = memo((props: Props) => {
	const { children, ...checkboxProps } = props;
	return <Checkbox {...checkboxProps}>{children}</Checkbox>;
});

```

Reply:
対応しました。コードが完結になって良いですね！覚えておきます！